service/src/main/java/uk/gov/hmcts/reform/rd/client/OrganisationApi.java

```diff
@@ -19,7 +19,8 @@
     OrganisationUsers findUsersByOrganisation(
         @RequestHeader(AUTHORIZATION) String authorisation,
         @RequestHeader(SERVICE_AUTHORIZATION) String serviceAuthorization,
-        @RequestParam(value = "status") Status status
+        @RequestParam(value = "status") Status status,
+        @RequestParam(value = "returnRoles") Boolean returnRoles
```

Comment:
Could we abstract this method to not expose this parameter? As easier Api for use would it be possible to expose client without this method and always pass false?

Reply:
not sure if we should, it is interface definition. We have parameter hidden at service layer